interface/main/tabs/templates/user_data_template.php

```diff
@@ -25,7 +25,7 @@
                 <?php
                   if ($userQuery['picture_url']) {
                     $picture_url = $userQuery['picture_url'];
-                    echo "<img src='../../../profile_pictures/$picture_url' height='64px' width='64px' style='border-radius: 40px;'></a>";
+                    echo ("<img src='../../../sites/".$_SESSION['site_id']."/profile_pictures/".$picture_url."' height='64px' width='64px' style='border-radius: 40px;'></a>");
```

Comment:
if globals.php is called, then use the global index for the sites directory, and not a relative path.

Reply:
Nevermind.  Looks like globals are not called here.